www/docs/further/further-reading.md

```diff
@@ -48,8 +52,9 @@ The key difference is that Blitz is a full-stack framework and tRPC is a library
 
 ### Benefits with Blitz.js
 
-- More batteries included and easier to setup
-- Blitz has a plenty of nice DX-benefits with it being a full-stack famework  
-- Blitz' custom build pipeline enables stuff like CMD+clicking a function to jump to the backend function  
-- Code scaffolding, plug-and-play auth, etc
+- First class data layer integrated with the Next.js runtime, resulting in a more seamless experience
+- If already using Next.js, it's extremely simple to switch to Blitz and adopt the data layer incrementally
+- Automatic integration with Blitz auth
+- Since you import server functions directly to the frontend, you can do things like CMD+click a function to jump to definition
 - Blitzjs has a bigger community  
+- Not a single API-endpoint so you don't have any code size issues in serverless environments
```

Comment:
Yep, I get that. If we remove this here, then I think we should remove the "Single API-endpoint" point from the trpc list.

Reply:
It is the default behaviour but can be opted out from. I'm not a big fan of SOA myself and I think people shouldn't do SOA unless they have very particular needs.

Having a single endpoint is an advantage for most apps as they rarely hit cold starts, get fewer connection pooling issues, enable batched requests, etc.

Conclusion - I'm not removing it, but it might be worth rephrasing that it's opt-outable.